src/main/java/com/cincialert/cincialert/dto/User.java

```diff
@@ -0,0 +1,18 @@
+package com.cincialert.cincialert.dto;
+
+import lombok.Data;
+
+@Data
+public class User {
+
+    /**
+     * The user's username, unique.
+     */
+    private String username;
```

Comment:
Adding an ID here would be beneficial, especially because the database could index the id column leading to more performant database queries.